components/mpas-ocean/src/shared/mpas_ocn_tracer_hmix_redi.F

```diff
@@ -416,22 +433,29 @@ subroutine ocn_tracer_hmix_Redi_tend(meshPool, layerThickness, layerThicknessEdg
          do k = minLevelCell(iCell), maxLevelCell(iCell)
             do iTr = 1, ntracers
                tend(iTr, k, iCell) = tend(iTr, k, iCell) + redi_term1(iTr, k, iCell)
-               tend(iTr, k, iCell) = tend(iTr, k, iCell) + rediKappaCell(iCell)*2.0_RKIND* &
-                                     (RediKappaSfcTaper(k, iCell)*RediKappaScaling(k, iCell)* &
-                                      redi_term3_topOfCell(iTr, k, iCell) - RediKappaSfcTaper(k + 1, iCell) &
-                                      *RediKappaScaling(k + 1, iCell)*redi_term3_topOfCell(iTr, k + 1, iCell))
             end do
          end do
 
-         do i = 1, nEdgesOnCell(iCell)
-            if (cellsOnCell(i, iCell) .eq. nCellsP1) cycle
-            iEdge = edgesOnCell(i, iCell)
-            do k = minLevelEdgeBot(iEdge), maxLevelEdgeTop(iEdge)
+         if (maxLevelCell(iCell) .ge. config_Redi_min_layers_diag_terms) then
```

Comment:
Yes, breaking them up here with the conditional outside is better.  And the config variables are fine to use directly. I've been copying some config variable into module variables because (1) some of the config names are just really really long and (2) if we go back to a private config with retrievals later, it would be better to retrieve once and make it a private module variable.